commitlint.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
This file should be removed. It was here when we were using commitlint as a hook, but we got rid of that when we went to squash merge PRs